.travis/build_ppc64le.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+# Start Echo API
+cd ..
+bundle exec rackup --port 3000 --host 0.0.0.0 --env production &
```

Comment:
Note to self - I think the job would fail here :-(

Reply:
Ok, guess we'll find out soon.

Reply:
My guess was right, I will push few changes on top of this.
`configuration /home/travis/build/3scale/config.ru not found`